src/monster.cpp

```diff
@@ -513,6 +511,28 @@ std::string monster::extended_description() const
                              enumerate_as_string( abilities ).c_str() ) << std::endl;
     }
 
+    std::vector<std::string> dangers;
+    if( type->has_flag( m_flag::MF_GRABS ) ) {
+        dangers.emplace_back( "grab" );
+    }
+    if( type->has_flag( m_flag::MF_VENOM ) ) {
+        dangers.emplace_back( "poison" );
+    }
+    if( type->has_flag( m_flag::MF_PARALYZE ) ) {
+        dangers.emplace_back( "paralyze" );
+    }
+    if( type->has_flag( m_flag::MF_BLEED ) ) {
+        dangers.emplace_back( "cause bleed" );
+    }
+    if( type->has_flag( m_flag::MF_ELECTRIC ) ) {
```

Comment:
Electric flag by itself does not cause the zapback - it only immunizes the monster against electricity.